webui/build.gradle

```diff
@@ -62,8 +62,6 @@ dependencies {
 
     implementation group: "com.sparkjava", name: "spark-core", version: spark_core_version
 
-    implementation group: "ch.unibas.dmi.dbis.polyphenydb", name: "polyphenydb-jdbc-driver", version: polyphenydb_jdbc_driver_version
```

Comment:
Has this been replaced by another dependency? Or is it simply no longer needed?

Reply:
This is already since quite some time not required anymore.